build/tag.js

```diff
@@ -66,11 +66,12 @@ function processImageData (data) {
       if (imageData) {
         tagImage(imageData.Id.split(':')[1], image, version);
         if (releases[version].lts) {
-          tagImage(imageData.Id.split(':')[1], image, 'lts');
+          if (version == versions[versions.length - 2]) {
```

Comment:
Won't this comparison always return the next to last result? What does that mean when Node.js 8.0 is released? Wouldn't v7 then end up being tagged as 'lts'? That wouldn't work. I think we need to find the largest version number where `releases[version].lts` is `true`.